zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateUserTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.client.command;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.client.api.command.ProblemException;
+import io.camunda.zeebe.it.util.ZeebeAssertHelper;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources.AutoCloseResource;
+import java.time.Duration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+@AutoCloseResources
+class CreateUserTest {
+
+  @AutoCloseResource ZeebeClient client;
+
+  @TestZeebe
+  private final TestStandaloneBroker zeebe = new TestStandaloneBroker().withRecordingExporter(true);
+
+  @BeforeEach
+  void initClientAndInstances() {
+    client = zeebe.newClientBuilder().defaultRequestTimeout(Duration.ofSeconds(15)).build();
+  }
+
+  @Test
+  void shouldCreateUser() {
+    // when
+    client
+        .newUserCreateCommand()
+        .username("username")
+        .name("name")
+        .email("email@example.com")
+        .password("password")
+        .send()
+        .join();
+
+    // then
+    ZeebeAssertHelper.assertUserCreated(
+        "username",
+        (user) -> {
+          assertThat(user.getEmail()).isEqualTo("email@example.com");
+          assertThat(user.getName()).isEqualTo("name");
+          assertThat(user.getPassword()).isNotNull();
+        });
+  }
+
+  @Test
+  void shouldRejectIfUsernameAlreadyExists() {
+    // given
+    client
+        .newUserCreateCommand()
+        .username("username")
+        .name("name")
+        .email("email@example.com")
+        .password("password")
+        .send()
+        .join();
+
+    // when / then
+    final var problem =
+        assertThatThrownBy(
+                () ->
+                    client
+                        .newUserCreateCommand()
+                        .username("username")
+                        .name("name")
+                        .email("email@example.com")
+                        .password("password")
+                        .send()
+                        .join())
+            .hasCauseInstanceOf(ProblemException.class)
+            .hasMessageContaining("Failed with code 400: 'Bad Request'")
```

Comment:
💭 IMO this should be a `409 Conflict` instead of a `400 Bad Request`. I've [raised this with the rest api team](https://camunda.slack.com/archives/C06UKS51QV9/p1724920818935409). Nothing you need to do about this, just wanted to share.

Reply:
@maryarm FYI a PR to change the status code of ALREADY_EXISTS is getting merged to main. So you'll have to modify this assertion to fix the CI on this one before you can merge the PR